### ISSUE TITLE
Removing prefix for saving signature

### ISSUE THREAD

**Issue by APusch-Tenno** [MarvinKlein1508/SignaturePad#48]
hi,

earlier you provided a solution on how to load existing Base64-signatures.
Now I have to save the signatures in the same format again, because another application is also accessing them.

It's probably an easy solution with converting and removing the ealier added prefix, but before I do some trial and error, I'm just asking someone, who has more insight into this.

Could you provide me with a solution or a quick hint on how to achieve this?

Thanks

**Comment by MarvinKlein1508** [comment]
If I assume that you have converted the signature with UTF8, then you can simply skip the first 23 characters of the byte array in the conversion like this:
```cs
string signature = Encoding.UTF8.GetString(Input.Signature[23..])
```

**Comment by APusch-Tenno** [comment]
Sorry, I couldn't reply earlier, but now I was able to test this and unfortunately a check of the Base64 string says, that the string has an incorrect length.

Luckily I just realized how the string usually starts and that there is an "i" missing, so the correct conversion should be:
`string signature = Encoding.UTF8.GetString(Input.Signature[22..])`

With this it's working fine.
Thanks for your help!
